### ISSUE TITLE
add google map component

### ISSUE THREAD

**Issue by stratospark** [stratospark/react-ss#5]

**Comment by stratospark** [comment]
I'm trying leaflet first. It seems to require some server-side integration to make it work. I've pushed a branch here: https://github.com/stratospark/react-ss/tree/maps

**Comment by stratospark** [comment]
Got Leaflet to work on the client side and placeholder rendered for server-side. Currently, it breaks if you switch away from the page.

@igaltivoni In your Plentyhood app, do you ever re-render the map? Is there ever a situation where you go to a view without a map then go back to a view with a map?

![image](https://cloud.githubusercontent.com/assets/93366/4803157/88d96aa6-5e52-11e4-82f1-b8a4b6e6f992.png)

**Comment by igalgh** [comment]
In Plentyhood.com you can switch between the map view and the splash
screen. Every time you go to the page which has leaflet, the map is
re-rendered.

Is this what you asked?

On Mon, Oct 27, 2014 at 8:30 PM, Patrick Rodriguez <notifications@github.com

**Comment by stratospark** [comment]
Yes. Was wondering if you cached the map object somewhere else and
reattached it when it became visible again. Probably premature optimization.

On Monday, October 27, 2014, igal notifications@github.com wrote:

> In Plentyhood.com you can switch between the map view and the splash
> screen. Every time you go to the page which has leaflet, the map is
> re-rendered.
> 
> Is this what you asked?
> 
> On Mon, Oct 27, 2014 at 8:30 PM, Patrick Rodriguez <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>
> 
> > wrote:
> > 
> > Got Leaflet to work on the client side and placeholder rendered for
> > server-side. Currently, it breaks if you switch away from the page.
> > 
> > @igaltivoni https://github.com/igaltivoni In your Plentyhood app, do
> > you ever re-render the map? Is there ever a situation where you go to a
> > view without a map then go back to a view with a map?
> > 
> > [image: image]
> > <
> > https://cloud.githubusercontent.com/assets/93366/4803157/88d96aa6-5e52-11e4-82f1-b8a4b6e6f992.png>
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/stratospark/react-ss/issues/5#issuecomment-60705360.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/stratospark/react-ss/issues/5#issuecomment-60710553.

## 

-pr
